src/android/FirebasePlugin.java

```diff
@@ -1280,13 +1281,30 @@ public void sendUserEmailVerification(final CallbackContext callbackContext, fin
             public void run() {
                 try {
                     FirebaseUser user = FirebaseAuth.getInstance().getCurrentUser();
+                    ActionCodeSettings actionCodeSettings = null;
+
+                    if(!args.isNull(0)) {
```

Comment:
@tzappia I had to define it outside the block because the check for actionCodeSettings below didn't know about the variable since it's defined in an if block.

Reply:
How about:
```
if(!args.isNull(0)) {
  JSONObject actionCodeSettingsParams = args.getJSONObject(0);
  actionCodeSettings = ActionCodeSettings.newBuilder()...
  handleTaskOutcome(user.sendEmailVerification(actionCodeSettings), callbackContext);
} else {
  handleTaskOutcome(user.sendEmailVerification(), callbackContext);
}
```